index.html

```diff
@@ -177,22 +179,22 @@ <h3 class="connect">LET’S CONNECT</h3>
                 </div>
                 <article class="right-content">
                     <h2 class="lang-info">Languages <span class="more-lang">></span></h2>
-                    <ul class="languages">
-                        <li>
-                            <img src="assets/images/js.png" alt="">
-                            <h4>Javascript</h4>
-                        </li>
-                        <li>
-                            <img src="assets/images/html.png" alt="">
-                            <h4>HTML</h4>
-                        </li>
-                        <li>
-                            <img src="assets/images/css.png" alt="">
-                            <h4>CSS</h4>
-                        </li>
-                    </ul>
 
                     <ul class="tools-container">
+                        <ul class="languages">
+                            <li>
+                                <img src="assets/images/js.png" alt="">
+                                <h4>Javascript</h4>
+                            </li>
+                            <li>
+                                <img src="assets/images/html.png" alt="">
+                                <h4>HTML</h4>
+                            </li>
+                            <li>
+                                <img src="assets/images/css.png" alt="">
+                                <h4>CSS</h4>
+                            </li>
+                        </ul>
                         <li class="tools">framworks <span class="more">></span></li>
                         <li class="tools">skills <span class="more">></span></li>
```

Comment:
- [ ] Please make sure that there are no typos on your project. Typos can make your project not look professional.